authors/apps/profiles/urls.py

```diff
@@ -1,9 +1,16 @@
 from django.urls import path
 from .views import (
     RetrieveUpdateProfileView,
-    ListProfileView)
+    ListProfileView,
+    FollowProfileView,
+    FollowersView,
+    FollowingView
+)
 
 urlpatterns = [
     path('profiles/<str:username>/', RetrieveUpdateProfileView.as_view()),
     path('profiles/', ListProfileView.as_view()),
+    path('profiles/<str:username>/follow/', FollowProfileView.as_view()),
```

Comment:
@NLSanyu , please use the convention our TTL told us to use for naming views.

Reply:
I have changed this. Thanks @Stanley-Okwii